### PR TITLE
6 new third parties and more subdomains

### PR DESCRIPTION
**Pull request by zeman**

### CODE REVIEW

**Comment by zeman** [comment]
We'd certainly prefer a `*.domain.com` approach to identifying a third party where appropriate rather than an explicit list of subdomains. As you point out, some third parties have dozens of numeric variations and it's painful adding all of those!

File: data/entities.json5 [hunk 1]
```diff
@@ -328,13 +362,13 @@
...
     "categories": ["analytics"],
-    "domains": ["idsync.rlcdn.com", "circulate.com"]
+    "domains": ["idsync.rlcdn.com", "circulate.com", "id.rlcdn.com", "api.rlcdn.com", "cdn.rlcdn.com"]
   },
   {
     "name": "Drawbridge",
     "homepage": "https://www.drawbridge.com/",
     "categories": ["ad"],
-    "domains": ["p.adsymptotic.com"]
+    "domains": ["p.adsymptotic.com", "pinterest.adsymptotic.com"]
```
[the remark this answers is not in the document]
Comment (zeman) [reply data/entities.json5]: Yep, really happy to loosen it off to `adsymptotic.com` and include all subdomains.

File: data/entities.json5 [hunk 2]
```diff
@@ -109,6 +119,7 @@
       "www.google-analytics.com",
       "ssl.google-analytics.com",
       "google-analytics.com",
+      "ampcid.google.com",
```
[the remark this answers is not in the document]
Comment (zeman) [reply data/entities.json5]: Hmm... doesn't feel right to me to be under the main AMP project. Instead I've moved it out to a new Google AMP service.

[the remark this answers is not in the document]
Comment (patrickhulce) [reply data/entities.json5]: Good thinking!

**Comment by patrickhulce** [comment]
> We'd certainly prefer a *.domain.com approach

Sounds good @zeman! Just to make sure you know, when the library is actually classifying a request it is already a using `*.domain.com` approach if the entity is the only one to lay claim to a eTLD+1 domain. The only change necessary would be if you're using the raw JSON instead of the library logic. 👍

**Comment by zeman** [comment]
@patrickhulce I'm continuing to add more third parties. As I do so should I start collapsing down the list of domains to eTLD or eTLD+1? For smaller third parties we'd prefer to just go with eTLD unless it's important to split out different service with the same eTLD eg. Google.

Here's an example for AddThis:

`"s7.addthis.com", "addthiscdn.com", "addthisedge.com", "r.dlx.addthis.com", "su.addthis.com", "x.dlx.addthis.com"`

We'd prefer to drop down to:

`"addthis.com", "addthiscdn.com", "addthisedge.com"`

It seems unnecessary to capture eTLD+1 for someone like AddThis and there's quite a bit of subdomain variation we see across our tests?

**Comment by patrickhulce** [comment]
Yes @zeman that'd be great! I was planning on doing a mass rewrite at some point but if you want to do some incremental PRs that'd be great :)

**Pull request merged**